_packages/api/test/api.bench.ts

```diff
@@ -0,0 +1,297 @@
+import {
+    API,
+    type Project,
+} from "@typescript/api";
+import {
+    type FileSystem,
+    type FileSystemEntries,
+} from "@typescript/api/fs";
+import {
+    type Node,
+    type SourceFile,
+    SyntaxKind,
+} from "@typescript/ast";
+import fs from "node:fs";
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import { Bench } from "tinybench";

```

Comment:
In a way I wish were just using vitest here (since it has tinybench built-in), but it's all pretty similar.